internal/cmd/cmd.go

```diff
@@ -40,67 +40,58 @@ This could be in indication that the ingress port is already in use by a differe
 The ingress port can be changed by passing the flag --port.`
 )
 
-// Execute adds all child commands to the root command and sets flags appropriately.
-// This is called by main.main(). It only needs to happen once to the rootCmd.
-func Execute(ctx context.Context, cmd *cobra.Command) {
-	if err := cmd.ExecuteContext(ctx); err != nil {
-		pterm.Error.Println(err)
-
-		switch {
-		case errors.Is(err, localerr.ErrAirbyteDir):
-			pterm.Println()
-			pterm.Info.Println(helpAirbyteDir)
-		case errors.Is(err, localerr.ErrDocker):
-			pterm.Println()
-			pterm.Info.Println(helpDocker)
-		case errors.Is(err, localerr.ErrKubernetes):
-			pterm.Println()
-			pterm.Info.Println(helpKubernetes)
-		case errors.Is(err, localerr.ErrIngress):
-			pterm.Println()
-			pterm.Info.Println(helpIngress)
-		case errors.Is(err, localerr.ErrPort):
-			pterm.Println()
-			pterm.Info.Printfln(helpPort)
-		}
-
-		os.Exit(1)
+func HandleErr(err error) {
+	if err == nil {
+		return
 	}
-}
 
-// NewCmd returns the abctl root cobra command.
-func NewCmd() *cobra.Command {
-	cobra.EnableTraverseRunHooks = true
+	pterm.Error.Println(err)
 
-	var (
-		flagVerbose bool
-	)
+	var errParse *kong.ParseError
+	if errors.As(err, &errParse) {
+		_ = kong.DefaultHelpPrinter(kong.HelpOptions{}, errParse.Context)
+	}
 
-	cmd := &cobra.Command{
-		Use:   "abctl",
-		Short: pterm.LightBlue("Airbyte") + "'s command line tool",
-		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-			if flagVerbose {
-				pterm.EnableDebugMessages()
-			}
+	switch {
+	case errors.Is(err, localerr.ErrAirbyteDir):
```

Comment:
💡Idea (not for this PR): It might be useful to define an `AbctlError` interface that wraps implements both the `Error` interface and also `Help` (which is an interface we can define that requires a `Help() string` method. That way we can define the errors and help messages in the same place.

The downside might be that it leads to implement finer-grained errors, which might not be ideal. Thoughts?